test/data/enrichment/test_service.py

```diff
@@ -76,13 +76,12 @@ def test_enrichment_query_by_points_one_variable(self):
         queries = _enrichment_queries(user_dataset, tablename, query_function, **kwargs)
 
         expected_queries = ['''SELECT data_table.enrichment_id,
-            CRMCYBURG,
-            ST_Area(enrichment_geo_table.geom) AS CRMCYBURG_area,
-            NULL AS CRMCYBURG_population
+            enrichment_table.CRMCYBURG,
+            ST_Area(enrichment_geo_table.geom) AS view_ags_demographics_crimerisk_usa_blockgroup_2015_yearly_2018_area
         FROM `carto-do-customers.{user_dataset}\
-            .ags_demographics_crimerisk_usa_blockgroup_2015_yearly_2018` enrichment_table
+            .view_ags_demographics_crimerisk_usa_blockgroup_2015_yearly_2018` enrichment_table
```

Comment:
I think we are trying to avoid using real names

Reply:
We need real names because of functions are using the real catalog, so we need real examples of table names. Also, I cannot see any problems, because we are offering publicly this dataset through website and catalog